app/services/ScanningResultHandler.scala

```diff
@@ -22,10 +22,14 @@ import model.S3ObjectLocation
 
 import scala.concurrent.{ExecutionContext, Future}
 
+sealed trait InstanceSafety
```

Comment:
I always add `extends Product with Serializable` to my sealed traits for reasons described here: https://underscore.io/blog/posts/2015/06/04/more-on-sealed.html